src/app/shared/stache/modules/code-block/code-block.component.ts

```diff
@@ -4,6 +4,7 @@ declare let Prism: any;
 import 'prismjs/prism';
 import 'prismjs/components/prism-typescript';
 import 'prismjs/plugins/normalize-whitespace/prism-normalize-whitespace';
+require('style-loader!prismjs/themes/prism.css');
```

Comment:
I tested this locally: place this `require` back in the code block module. Also include it above the `require('style-loader!./app.scss');` in the app-extras file. That way, we can preserve the location of the css file in the module, but also implement it for stache SPA. (This is how we're handling the smoothscroll polyfill as well.)